iModelCore/ECDb/PublicAPI/ECDb/ECSqlStatement.h

```diff
@@ -461,6 +461,15 @@ struct EXPORT_VTABLE_ATTRIBUTE ECSqlStatement
         //! - @p columnIndex is out of bounds
         ECDB_EXPORT IECSqlValue const& GetValue(int columnIndex) const;
 
+        //! Serialize ECSQL statement row to JSON
+        //! @param[in] abbreviateBlobs abbreviate blobs as a Json string with info about the blob
+        //! @param[in] classIdToClassNames write the fully qualified name of an ECClass into a JSON value: {schema name}.{class name}
+        //! @param[in] useJsName use Js property name
+        //! @param[in] includeMetaData include metadata in the JSON output
+        //! @param[out] outJson ECSQL statement row to Serialize as JSON

```

Comment:
I believe elsewhere the param arguments are documented in a positional order.

Reply:
You're right. I'll update the documentation here to maintain consistency.